projects/addon-editor/directives/tiptap-editor/tiptap-editor.service.ts

```diff
@@ -314,9 +314,17 @@ export class TuiTiptapEditorService extends AbstractTuiEditor {
     }
 
     setValue(value: string): void {
-        if (value !== this.html) {
```

Comment:
If we move this logic back here, we can skip adding public method of clearing history. It's better not to extend public API unless we have to.